src/EditorFeatures/CSharpTest/EditAndContinue/StatementMatchingTests.cs

```diff
@@ -863,6 +863,323 @@ public void LocalFunctionDefinitions()
+                { "var e = from q in a.Where(condition) select q + 1", "var e = from q in a.Where(l => l < 0) select q + 1" },
+                { "e = from q in a.Where(condition) select q + 1", "e = from q in a.Where(l => l < 0) select q + 1" },
+                { "from q in a.Where(condition)", "from q in a.Where(l => l < 0)" },
+                { "select q + 1", "select q + 1" },  // select clause
+                { "select q + 1", "select q + 1" }   // query body
+            };
+
+            expected.AssertEqual(actual);
+        }
+
+        [Fact]
+        public void LocalFunctions5()
+        {
+            var src1 = @"
+int a() 
+{
+    int b()
+    {
+        int c()
+        {
+            int d() { return 0; };
+        };
+
+        return c();
+    };
+
+    return b();
+}
+";
+            var src2 = @"
+int a() 
+{
+    int b()
+    {
+        int c()
+        {
+            int d() { return 0; };
+        };
+
+        return c();
+    };
+
+    return b();
+}
+";
+
+            var matches = GetMethodMatches(src1, src2);
+            var actual = ToMatchingPairs(matches);
+
+            var expected = new MatchingPairs
+            {
+                { "int a()  {     int b()     {         int c()         {             int d() { return 0; };         };          return c();     };      return b(); }", "int a()  {     int b()     {         int c()         {             int d() { return 0; };         };          return c();     };      return b(); }" },
+                { "{     int b()     {         int c()         {             int d() { return 0; };         };          return c();     };      return b(); }", "{     int b()     {         int c()         {             int d() { return 0; };         };          return c();     };      return b(); }" },
+                { "int b()     {         int c()         {             int d() { return 0; };         };          return c();     }", "int b()     {         int c()         {             int d() { return 0; };         };          return c();     }" },
+                { "{         int c()         {             int d() { return 0; };         };          return c();     }", "{         int c()         {             int d() { return 0; };         };          return c();     }" },
+                { "int c()         {             int d() { return 0; };         }", "int c()         {             int d() { return 0; };         }" },
+                { "{             int d() { return 0; };         }", "{             int d() { return 0; };         }" },
+                { "int d() { return 0; }", "int d() { return 0; }" },
+                { "{ return 0; }", "{ return 0; }" },
+                { "return 0;", "return 0;" },
+                { "return c();", "return c();" },
+                { "return b();", "return b();" }
+            };
+
+            expected.AssertEqual(actual);
+        }
+
+        [Fact]
+        public void LocalFunctions6()
```

Comment:
OK. Let us remove it.

---
In reply to: [132338373](https://github.com/dotnet/roslyn/pull/21396#discussion_r132338373) [](ancestors = 132338373)